### PR TITLE
add logout feature

### LINKED ISSUE #2: implement logout functionality

### PR DESCRIPTION
**Pull request by kasthurimahadeva**
closes #2

### CODE REVIEW

File: src/main/resources/templates/authorize.html [hunk 1]
```diff
@@ -5,6 +5,16 @@
...
+<div>
+    Logged in user: <b th:inline="text" class="user"> [[${#httpServletRequest.remoteUser}]] </b>
+    <div>
+        <br>
+        <form th:action="@{/logout}" method="POST">
+            <input type="submit" value="Logout"/>
+        </form>
+        <br>
+    </div>
+</div>
```
Comment (KajanM) [inline src/main/resources/templates/authorize.html]: since this is repeating, better to add as a fragment, anyway it's ok for now.

Comment (kasthurimahadeva) [reply src/main/resources/templates/authorize.html]: Okay, I 'll address that in a separate PR

Comment (KajanM) [reply src/main/resources/templates/authorize.html]: thanks

**Comment by KajanM** [comment]
thanks

**Pull request merged**